docs/versioned_docs/version-zenith/quickstart/822194-daggerize.mdx

```diff
@@ -0,0 +1,234 @@
+
+    List available functions:
+
+    ```shell
+    dagger -m hello functions
+    ```
+
+    You should see a familiar result:
+
+    ```shell
+    Name    Description
+    hello   Say hello to the world!
+    ```
+
+    Call a function:
+
+    ```shell
+    dagger -m hello call hello
+    ```
+
+    Again, the output should be familiar:
+
+    ```shell
+    hello, world!
+    ```
+
+5. The `hello` module is a good example to start with, but it doesn't have any functions to work with a Go project. To Daggerize the Go build for this example project, install the Go builder module from a previous section as another dependency:
+
+    ```shell
+    dagger install github.com/kpenfound/dagger-modules/golang
+    ```
+
+    Then, call the `Build()` function from the Go builder module to build the project and chain other function calls to export the built binary to the host:
+
+    ```shell
+    dagger -m golang --proj . call \
+      # call the go builder function
+      build --os=$(uname -s) --arch=$(uname -m) --args . \
+      # select the binary file produced by the build
+      file --path=./greetings-api \
+      # export
+      export --path=./api-server
+    ```
+
+    Test the newly-built binary by starting the service and sending it a request:
+
+    ```shell
+    ./api-server &
+    curl localhost:8080
+    ```
+
+    You should see the following output:
+
+    ```shell
+    got / request from 127.0.0.1:51418
+    {"greeting":"Hello Kubecon!"}
+    ```
+
+6. At this point, you've partly Daggerized your project, to the extent of building a binary with Dagger. Next, how about containerizing and publishing it?
+
+An easy way to do this is to first add the Wolfi container builder module from earlier as another dependency:
+
+    ```shell
+    dagger install github.com/shykes/daggerverse/wolfi
+    ```
+
+    Then, add the built binary from the previous step to a Wolfi base container image, and publish it:
+
+    ```shell
+    dagger -m wolfi call \
+      # build a Wolfi container image
+      # add the binary file
+      container with-file --path=/usr/local/bin/api-server --source=./api-server \
+      # publish the container image
+      publish --address ttl.sh/my-api-server
+    ```
+
+    You should see the address of the container image after it's built and published:
+
+    ```shell
+    ttl.sh/my-api-server@sha256:0f9b2642d3b137aac72120ba38ecf56df3ddfd5d6815b0e3fc3ede0585686ce8
+    ```
+
+    Start a new container from the published container image, expose the service on host post 8081, and test the service using the commands below:
+
+    ```shell
+    docker run \
+      --name ctr \
+      --rm \
+      -d \
+      -p 8081:8080 \
+      -it ttl.sh/my-api-server /usr/local/bin/api-server
+    curl localhost:8081
+    docker stop ctr
+    ```
+
+    You should see the following output:
+
+    ```shell
+    {"greeting":"Hello Kubecon!"}
```

Comment:
@kpenfound could you please update this output in the source project? Maybe `Hello from Dagger!`